meson.build

```diff
@@ -61,6 +61,7 @@ version_gen_h = vcs_tag(
 subdir('libfdt')
 
 dtc_tools = []
+util_dep = declare_dependency()
```

Comment:
I tried to build this change, but I get a build error because `util.h` isn't defined.
```
In file included from ../tests/testutils.c:36:
../tests/tests.h:128:10: fatal error: util.h: No such file or directory
  128 | #include "util.h"
      |          ^~~~~~~~
```

Instead, we can move the entire `util_dep` declaration here.

Reply:
yes, that works